web/browsersupport/README.md

```diff
@@ -0,0 +1,13 @@
+# [Engineering Playbook](../../README.md) / [Web Development](../README.md) / Browser Support
+
+## Overview
+Unless specifically negotiated with the client, we generally support only the latest browser versions supported by operating systems.
```

Comment:
Milmove ADR: https://github.com/transcom/mymove/blob/master/docs/adr/0016-Browser-Support.md#browser-support-for-prototype

This PR is a WIP because not sure how applicable those are to our general practices.